packages/constants/index.js

```diff
@@ -35,4 +35,5 @@ export const GRAPHQL_TARGETS = {
   DASHBOARD_SERVICE: 'dashboard',
   PIM_INDEXER: 'pim-indexer',
   SETTINGS_SERVICE: 'settings',
+  CORE_WS: 'core-ws',
```

Comment:
Thought about the name literally ten minutes. `core` just sounds really silly. We can also name it something completely different but I don't know how much sense that makes.

Reply:
Can't think on another name since this is the one that I've been reading this past weeks. So to me it's fine 😅